### PR TITLE
feat: refactor system stats and personal balance sections

### LINKED ISSUE #212: Update the UI of system stats section
Add new [fields](https://babylonlabs.atlassian.net/wiki/spaces/BABYLON/pages/41877853/Phase-2+UI+Components#1.-System-Statistics) and use new `/v2/stats` [endpoint](https://babylonlabs.atlassian.net/wiki/spaces/BABYLON/pages/40239375/Schemas#Endpoints)

As a part of this task we should also replace our stats provider `/src/app/context/api/StakingStatsProvider.tsx` with a custom hook on top of `useApiQuery`

### PR DESCRIPTION
**Pull request by totraev**
Just a static UI components without any API integration

### Before
Screenshot 2024-10-16 at 13 48 55

### After
Screenshot 2024-10-16 at 13 51 05

Closes #212 #214

### CODE REVIEW

**Review by gbarkhatov — APPROVED** [review]
Are we okay with left-aligned content and header there? https://i.imgur.com/47YZRxV.png

**Comment by totraev** [comment]
> Are we okay with left-aligned content and header there? https://i.imgur.com/47YZRxV.png

@jrwbabylonlab WDYT think? I don't have strong opinion on UI/UX

File: src/app/hooks/api/useVersions.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { getGlobalParams } from "@/app/api/getGlobalParams";
+import { useAPIQuery } from "@/app/hooks/api/useApi";
+
+export const VERSIONS_KEY = "VERSIONS";
```
[the remark this answers is not in the document]
Comment (totraev) [reply src/app/hooks/api/useVersions.ts:4]: I don't think we will have `react-query` in this project in the future

File: src/app/components/PersonalBalance/PersonalBalance.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { BalanceItem } from "./BalanceItem";
+
+export function PersonalBalance() {
```
[the remark this answers is not in the document]
Comment (totraev) [reply src/app/components/PersonalBalance/PersonalBalance.tsx:3]: @jrwbabylonlab without Tomo integration it doesn't really make sense

File: src/app/hooks/api/useVersions.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (totraev) [reply src/app/hooks/api/useVersions.ts:4]: @jrwbabylonlab https://github.com/babylonlabs-io/simple-staking/issues/111

**Pull request merged**